### ISSUE TITLE
Should use single space after period to follow Go style.

### ISSUE THREAD

**Issue by shurcooL** [google/go-github#548]
The Go project tree used to be pretty inconsistent about using two vs one spaces between sentences. However, eventually an agreement came to be to prefer single space, and the following CLs fixed up the rest of the inconsistencies:

- https://go-review.googlesource.com/c/20022/
- https://go-review.googlesource.com/c/32878/
- https://go-review.googlesource.com/c/32879/

I wrote a entry for this at https://dmitri.shuralyov.com/idiomatic-go#single-spaces-between-sentences.

This is a large and popular Go library, and I believe we should be following Go style whenever possible, and not diverge without good reasons. It'll be easier for people who contribute because they won't have to remember that the Go project uses single spaces, while here we use both single and double spaces.

Fixing this is easy (and can be automatically generated, for the most part), but is going to touch a lot of lines. I made #549.

/cc @willnorris, I want to get your approval because I understand you prefer to use double spaces, since that's how the project was started.

**Comment by shurcooL** [comment]
@willnorris if you have a few minutes, mind letting me know if this is okay by you? I'd prefer not to let the [PR](https://github.com/google/go-github/pull/549) go stale since it touches many lines of code and might cause conflicts in the future, and I'd also prefer not to outright merge it without hearing your thoughts (but given @gmlewis has LGTMed it, I'll probably have to do that if I don't hear back within a few days).

The TLDR is: are you okay with us following Go style of having a single space between sentences rather than two?
